Makefile

```
@@ -81,7 +81,7 @@ spacenet: eudico lotus-seed lotus-keygen lotus-shed
 spacenet-test: GOFLAGS+=-tags=spacenet
 spacenet-test:
 	export GOLOG_LOG_LEVEL="ERROR,mir-consensus=debug,mir-manager=error" MIR_INTERCEPTOR_OUTPUT="/tmp/mir-logs-`date +%s`" && echo "Interceptor output: $$MIR_INTERCEPTOR_OUTPUT"; \
-	go test $(GOFLAGS) -shuffle=on -v -count=1 -timeout=30m ./itests/mir_test.go
+	go test $(GOFLAGS) -shuffle=on -v -count=1 -timeout=30m -run TestMirWith3NodesOneByOneReconfiguration ./itests/mir_test.go
```

This has been left behind, right? 
```suggestion
	go test $(GOFLAGS) -shuffle=on -v -count=1 -timeout=30m ./itests/mir_test.go
```

---

This is WiP PR, and this test fails.